CMakeLists.txt

```diff
@@ -493,6 +493,9 @@ set(LLVM_LIT_ARGS "-sv")
 # LLVM_RUNTIME_OUTPUT_INTDIR to our own build dir.
 set(LLVM_LIT_OUTPUT_DIR ${LLVM_BUILD_DIR}/${CMAKE_CFG_INTDIR}/bin)
 
+# Currently, when cross compiling to ARM on Windows host, we don't build some of the tools required for running the tests.
```

Comment:
Does the target actually need to be conditionally defined? Is there harm in having a test target that doesn't yet pass on a particular platform?